### ISSUE TITLE
Microsoft.Toolkit.Mvvm no longer works with TunnelVisionLabs.ReferenceAssemblyAnnotator

### ISSUE THREAD

**Issue by chucker** [CommunityToolkit/dotnet#95]
### Describe the bug

I have a .NET Standard 2.0 project that references both this package, and a third-party one:

```xml
<PackageReference Include="Microsoft.Toolkit.Mvvm" Version="7.0.1" />
<PackageReference Include="TunnelVisionLabs.ReferenceAssemblyAnnotator" Version="1.0.0-alpha.160" PrivateAssets="all" />
<PackageDownload Include="Microsoft.NETCore.App.Ref" Version="[5.0.0]" />
```

If I upgrade Microsoft.Toolkit.Mvvm to 7.1.0 (or 7.1.2), I get multiple build errors:

```
(my project)/Microsoft.Toolkit.Mvvm.SourceGenerators/Microsoft.Toolkit.Mvvm.SourceGenerators.NullabilityAttributesGenerator/System.Diagnostics.CodeAnalysis.NotNullAttribute.cs(27,27): Error CS0101: The namespace 'System.Diagnostics.CodeAnalysis' already contains a definition for 'NotNullAttribute' (CS0101) (AltNetworkUtility)
```

This is likely because `TunnelVisionLabs.ReferenceAssemblyAnnotator` synthesizes the `NotNullAttribute`, and I'm _guessing_ recent versions of `Microsoft.Toolkit.Mvvm.SourceGenerators` do the same. This may be related to CommunityToolkit/WindowsCommunityToolkit#4168, although I couldn't quite find the offending line.

### Regression

7.0.1

### Steps to reproduce

In an empty folder, create a `csproj` like so:

```xml
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <TargetFramework>netstandard2.0</TargetFramework>
  </PropertyGroup>

  <ItemGroup>
      <PackageReference Include="Microsoft.Toolkit.Mvvm" Version="7.1.0" />
      <PackageReference Include="TunnelVisionLabs.ReferenceAssemblyAnnotator" Version="1.0.0-alpha.160" PrivateAssets="all" />
      <PackageDownload Include="Microsoft.NETCore.App.Ref" Version="[5.0.0]" />
  </ItemGroup>

</Project>
```

`dotnet build will` fail.

Modify the package reference to be `7.0.1`:

```xml
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <TargetFramework>netstandard2.0</TargetFramework>
  </PropertyGroup>

  <ItemGroup>
      <PackageReference Include="Microsoft.Toolkit.Mvvm" Version="7.1.0" />
      <PackageReference Include="TunnelVisionLabs.ReferenceAssemblyAnnotator" Version="1.0.0-alpha.160" PrivateAssets="all" />
      <PackageDownload Include="Microsoft.NETCore.App.Ref" Version="[5.0.0]" />
  </ItemGroup>

</Project>
```

The project will now build.

### Expected behavior

`Microsoft.Toolkit.Mvvm` should ideally detect if the attributes already exist, instead of creating them a second time.

### Other SDK version

.NET SDK 6.0.101

### Nuget packages

Microsoft.Toolkit.Mvvm 7.1.2

### Help us help you

Yes, but only if others can assist.

**Comment by Sergio0694** [comment]
Hey @chucker, thank you for the report!
I think this has hopefully been fixed by https://github.com/CommunityToolkit/dotnet/pull/89.
Could you try out a preview package from our nightly feed and see if that resolves the issue for you?
That'd be `CommunityToolkit.Mvvm 8.0.0-build.xxx` from `https://pkgs.dev.azure.com/dotnet/CommunityToolkit/_packaging/CommunityToolkit-MainLatest/nuget/v3/index.json`.

**Comment by chucker** [comment]
Hi @Sergio0694, yep! Looks like I can no longer reproduce the issue in `CommunityToolkit.Mvvm` version `8.0.0-build.10`.

Is this going to be backported to 7.x?

**Comment by Sergio0694** [comment]
Hi @chucker, unfortunately not, all code has been migrated to this repo and new updates will only be rolled out from here going forward, and they'll be published under the new `CommunityToolkit.*` alias on NuGet. If it helps, we've just published a new 8.0.0 Preview 1 release on NuGet, maybe it'd be feasible for you to switch to that instead, even though it's marked as pre-release?

Closing this issue in the meantime as it's resolved, but feel free to reply in here anyway! 🙂
